src/ui/entry/style.css

```
@@ -20,6 +20,7 @@
 @import url("../styles/config.css");
 @import url("../styles/_world-tabs.css");
 @import url("../styles/_account-pages.css");
+@import url("../styles/_cards.css");
```

Stylelint (17.14.0)

[error] Expected "url("../styles/_cards.css")" to be ""../styles/_cards.css"" (import-notation)

(import-notation)